pkgs/by-name/wa/wasm-language-tools/package.nix

```diff
@@ -0,0 +1,38 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitHub,
+  versionCheckHook,
+  nix-update-script,
+}:
+
+rustPlatform.buildRustPackage rec {
```

Comment:
Consider using `finalAttrs`

Reply:
I'd rather not as it increases eval times. But if you insist, I will

Reply:
Have not thought about that aspect